doc/user-docs/index.md

```diff
@@ -87,4 +87,12 @@ For an introduction to the various settings, see [Using Json Settings](UsingJson
 2. Terminal zoom can be changed by holding <kbd>Ctrl</kbd> and scrolling with mouse.
 3. Background opacity can be changed by holding <kbd>Ctrl</kbd>+<kbd>Shift</kbd> and scrolling with mouse. Note that acrylic transparency is limited by the OS only to focused windows.
 4. Open Windows Terminal in current directory by typing `wt -d .` in the address bar.
-5. Please add more Tips and Tricks.
+5. Random theme in terminal by using Zsh(Z-shell) [LearnMoreAboutZ-shell](https://github.com/ohmyzsh/ohmyzsh)
+* `sudo apt install zsh` 
+* `sh -c "$(curl -fsSL https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"`
+* `sudo apt-get install fonts-powerline`
+* `code ~/.zshrc` # to customize it
```

Comment:
Also change the word `code` to vim or nano. (I personally prefer vim.)

Reply:
I changed to vim but I use mostly vscode for my Web development thing and I  ❤ vscode .that's all about work and perspective😊